config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -80,36 +80,23 @@
   'Referrer-Policy' => 'strict-origin-when-cross-origin'
 }
 
-#
-# Todo =========================================================
-#
-
-# Change the digest class for the key generators to `OpenSSL::Digest::SHA256`.
-# Changing this default means invalidate all encrypted messages generated by
-# your application and, all the encrypted cookies. Only change this after you
-# rotated all the messages using the key rotator.
-#
-# See upgrading guide for more information on how to build a rotator.
-# https://guides.rubyonrails.org/v7.0/upgrading_ruby_on_rails.html
-# Rails.application.config.active_support.key_generator_hash_digest_class = OpenSSL::Digest::SHA256
-
 # Change the digest class for ActiveSupport::Digest.
 # Changing this default means that for example Etags change and
 # various cache keys leading to cache invalidation.
-# Rails.application.config.active_support.hash_digest_class = OpenSSL::Digest::SHA256
+Rails.application.config.active_support.hash_digest_class = OpenSSL::Digest::SHA256
 
 # Specifies whether generated namespaced UUIDs follow the RFC 4122 standard for namespace IDs provided as a
 # `String` to `Digest::UUID.uuid_v3` or `Digest::UUID.uuid_v5` method calls.
 #
 # See https://guides.rubyonrails.org/configuring.html#config-active-support-use-rfc4122-namespaced-uuids for
 # more information.
-# Rails.application.config.active_support.use_rfc4122_namespaced_uuids = true
-
-# ** Please read carefully, this must be configured in config/application.rb **
-# Change the format of the cache entry.
-# Changing this default means that all new cache entries added to the cache
-# will have a different format that is not supported by Rails 6.1 applications.
-# Only change this value after your application is fully deployed to Rails 7.0
-# and you have no plans to rollback.
-# When you're ready to change format, add this to `config/application.rb` (NOT this file):
-#  config.active_support.cache_format_version = 7.0
+Rails.application.config.active_support.use_rfc4122_namespaced_uuids = true
+
+# Change the digest class for the key generators to `OpenSSL::Digest::SHA256`.
+# Changing this default means invalidate all encrypted messages generated by
+# your application and, all the encrypted cookies. Only change this after you
+# rotated all the messages using the key rotator.
+#
+# See upgrading guide for more information on how to build a rotator.
+# https://guides.rubyonrails.org/v7.0/upgrading_ruby_on_rails.html
+Rails.application.config.active_support.key_generator_hash_digest_class = OpenSSL::Digest::SHA256
```

Comment:
Are there places we use encryption? There's none that are jumping out at me, but I might not be looking in the right spot. 

Cookies seem like one of the main possibilities but I can't see any calls to `cookies.encrypted` anywhere in the app

The example for cookies specifically show [here](https://guides.rubyonrails.org/v7.0/upgrading_ruby_on_rails.html#key-generator-digest-class-changing-to-use-sha256) wouldn't be hard to add, but better not to if we don't need it

cc @KevinMulhern @CouchofTomato would love your thoughts on this, I believe this is the only thing left stopping us from switching over to Rails 7 framework defaults

Reply:
I ran a brief test on the review app before and after toggling this setting on. I didn't notice any major issues, but this would likely be something that would be a bigger issue in a real environment (if it was an issue at all)

Reply:
The only impact I can think of is the devise cookies being invalidated, all users will be signed out.

But thats a not blocker, we already require users to sign in again after two weeks with our expiry policy. Just something to keep in mind as expected if we get questions in Discord about users being forced signed out after this goes out.